src/app/file-upload/file-upload.component.html

```diff
@@ -1,6 +1,8 @@
 <div class="files-info" *ngIf="showInfo">
-  Maximum of {{maxFiles}} files, up to {{maxSize}}MB each.
-  <br> Acceptable file types: {{fileExt.slice().join(', ')}}
+  <ng-container *ngIf="maxFiles > 0">Maximum of {{maxFiles}} files. </ng-container>
```

Comment:
I think showInfo is still relevant, because you might not want to show anything even if the values are valid.  Alternatively, you might want to show the details, but not the ones that aren't being enforced.

Yeah file size wasn't shown before, but it was still being enforced. So it seemed like it would make sense to show it if is a restriction.  But maybe there was a reason to not display it?

Reply:
OK.